src/i18n/translation.fr.js

```diff
@@ -5,6 +5,12 @@ export default {
     changeAudience: 'Refaire le questionnaire',
     audiencePrefix: "Les informations ci-dessous s'appliquent à "
   },
+  share: {
+    viewResults: 'Voir vos résultats partageables',
```

Comment:
Personlisé? I feel like that's the feature we wanna advertise, not the shareability

Reply:
cc @vanessajones